bridge/client/webrtc.js

```diff
@@ -795,8 +806,12 @@
                 if (!mdesc.ice) {
                     console.warn("setRemoteDescription: m-line " + i +
                         " is missing ICE credentials");
-                    mdesc.ice = {};
+                    mdesc.ice = {
+                        "iceOptions": {}
+                    };
                 }
+                if (mdesc.ice.iceOptions.trickle)
+                    canTrickle = true;
```

Comment:
Sounds strange, but this is how it is currently specced out in JSEP. It will become a session level attribute most likely.